rust-bindings/src/auto/bootconfig_parser.rs

```diff
@@ -26,14 +26,14 @@ impl BootconfigParser {
 
     #[doc(alias = "ostree_bootconfig_parser_clone")]
 #[must_use]
-    pub fn clone(&self) -> Option<BootconfigParser> {
+    pub fn clone(&self) -> BootconfigParser {
         unsafe {
             from_glib_full(ffi::ostree_bootconfig_parser_clone(self.to_glib_none().0))
         }
     }
 
     #[doc(alias = "ostree_bootconfig_parser_get")]
-    pub fn get(&self, key: &str) -> Option<glib::GString> {
+    pub fn get(&self, key: &str) -> glib::GString {
```

Comment:
This should probably stay nullable and you can use something like `nullable_return_is_error = "The key doesn't exists"`

Reply:
And thanks so much for spotting this!!